test/rulesets/Base/base.jl

```diff
@@ -53,24 +53,38 @@
         end
         @testset "Multivariate" begin
             x, y = rand(2)
-            ratan = atan(x, y) # https://en.wikipedia.org/wiki/Atan2
-            u = x^2 + y^2
-            datan = y/u - 2x/u
-            r, df = frule(atan, x, y)
-            @test r === ratan
-            @test df(1, 2) === datan
-            r, (df1, df2) = rrule(atan, x, y)
-            @test r === ratan
-            @test df1(1) + df2(2) === datan
-
-            rsincos = sincos(x)
-            dsincos = cos(x) - 2sin(x)
-            r, (df1, df2) = frule(sincos, x)
-            @test r === rsincos
-            @test df1(1) + df2(2) === dsincos
-            r, df = rrule(sincos, x)
-            @test r === rsincos
-            @test df(1, 2) === dsincos
+            @testset "atan2" begin
+                # https://en.wikipedia.org/wiki/Atan2
+                ratan = atan(x, y) # https://en.wikipedia.org/wiki/Atan2
```

Comment:
already on the line above
```suggestion
                ratan = atan(x, y)
```